app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,11 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+
+    <!--    my colors-->
```

Comment:
usually it is better practice to avoid such comments.

Reply:
Ok